### PR TITLE
Add fiscal sponsor projects to the site

### PR DESCRIPTION
**Pull request by NPDebs**
This PR fixes issue #869.
OLS's index fiscally sponsored community, RSE-AUNZ, has been added to the appropriate page (i.e [.../open-incubator/fiscal-hosting/](https://we-are-ols.org/open-incubator/fiscal-hosting/)).

**Changes made:**
- Created a new `fiscal_communities` array within the `_data/community.yaml` file and added data about RSE-AUNZ.
- Wrote logic to add RSE-AUNZ to the bottom of the fiscal-hosting page.

**Screenshot:**
Screenshot (752)

**Note for reviewers:**
- Does this count as mvp or robust structure (as requested in issue #870)?

Thanks for the review! 

## FOR CONTRIBUTOR

* [x] I have read the [CONTRIBUTING.md](https://github.com/open-life-science/open-life-science.github.io/blob/main/CONTRIBUTING.md) document

PR Type: 
* [ ] This PR adds a new blog post
* [x] This PR does something else (explain above)

 
## FOR REVIEWERS

Thanks for taking the time to review! :heart:

Here are the list of things to make sure of:
* [x] The website builds (a check will fail if not)
* [x] All images have been added within the Pull Request and they have Alt text
* [ ] If there are paragraphs or text, the key messages are highlighted
* [ ] All internal links (within OLS website) use the [`{% link path_to_file.md %}` format](https://jekyllrb.com/docs/liquid/tags/#link)
* [x] The preview corresponds to the changes described in the Pull Request
* [x] The code is tidy and passes the linting tests

### CODE REVIEW

File: open-incubator.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ streams:
   -
     title: Fiscal Hosting 
     description: |
-      We are helping early stage organizations with fiscal hosting. If you are interested, please contact directors at we-are-ols.org to discuss. 
+      We are helping early stage organizations with fiscal hosting. If you are interested, please contact directors at [team[at]we-are-ols.org](mailto:team@we-are-ols.org) to discuss. Read more about this on our [fiscal hosting page](/open-incubator/fiscal-hosting/).
```
[the remark this answers is not in the document]
Comment (NPDebs) [reply open-incubator.md:44]: Yeah. It typically doesn't work in the yaml frontmatter.

File: oryx-build-commands.txt [hunk 2]
```diff
@@ -0,0 +1,2 @@
+PlatformWithVersion=Python
```
[the remark this answers is not in the document]
Comment (NPDebs) [reply oryx-build-commands.txt:1]: It's from when I built the site with GitHub Codespaces. Clearly shouldn't be in this PR. 😅

[the remark this answers is not in the document]
Comment (NPDebs) [reply oryx-build-commands.txt]: Deleted. Thanks!

**Review by yochannah — APPROVED** [review]
visually the change looks good - I'll merge but leave the tag to note that Bérénice may wish to review further.

**Pull request merged**